### PR TITLE
Improve project setup

### PR DESCRIPTION
**Pull request by renatodelpupo**
Initially proposed during #19, split to be available sooner.

Updates:
- Node: Documents suggested version, also setting nvm up.
- NPM: Specify public registry, avoiding downloading private PWMB packages
- Index.html: Just applies linting

### CODE REVIEW

File: react-web-app/public/index.html [hunk 1]
```diff
@@ -1,44 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```
Comment (lukevance) [inline react-web-app/public/index.html:3]: Nice cleanup 🧹

File: react-web-app/package.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "react-web-app-trustly-sample",
   "version": "0.1.0",
+  "engines": {
+    "node": ">=20"
```
Comment (lukevance) [inline react-web-app/package.json:5]: can we not support a lower version like 18 or 16 even? No big deal if not, but lower versions are still very common so it'd be nice not to force people to update if it's not necessary

**Review by lukevance — APPROVED** [review]
LGTM 😎

File: react-web-app/package.json (hunk 2 above)
Comment (renatodelpupo) [reply react-web-app/package.json:5]: Node 16 should compile the project fine today, we don't have any breaking features so far.

My idea was to look forward by suggesting v20, but we can set the v16 as standard, no problem.

Comment (renatodelpupo) [reply react-web-app/package.json]: Node version suggestion updated to the minimum version required ✅

**Review by pedroabdenor — APPROVED** [review]
👍

**Review by andresguedes — APPROVED** [review]

**Pull request merged**